### DISCUSSION TITLE
Wake from sleep

### DISCUSSION THREAD

**Discussion by quenthal** [Blackymas/NSPanel_HA_Blueprint#499] [Ideas]
With version 3.2 it seems the way to wake from sleep is double tap on screen. Could this be changed/configured to single tap/swipe instead?

**Comment by Blackymas** [comment]
https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/493

**Comment by edwardtfn** [comment]
Duplicated with #554
Already fixed in v3.2.2
